### PR TITLE
testsuite: add README for rust bridged test

### PR DESCRIPTION
**Pull request by Riolku**

### CODE REVIEW

**Review by Xyene — APPROVED** [review]
LGTM, thanks!

File: testsuite/rust-bridged/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This test checks the shared target functionality of Rust, by having a submission and checker in Rust.
+
+The first case always runs fine, since the checker will not be compiled until after the submission is run. The second case will IR if the submission and checker have misconfigured target directories.
```
Comment (quantum5) [inline testsuite/rust-bridged/README.md:3]: Can we `M-q` this? Feels too long.

Comment (Riolku) [reply testsuite/rust-bridged/README.md:3]: What is `M-q`?

File: testsuite/rust-bridged/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+This test checks the shared target functionality of Rust, by having a submission and checker in Rust.
+
+The first case always runs fine, since the checker will not be compiled until after the submission is run.
+
+The second case will IR if the submission and checker have misconfigured target directories.
```
Comment (quantum5) [inline testsuite/rust-bridged/README.md:5]: In vim speak, what I want is `gggqG`.

(Break lines at 79 columns even if it breaks a sentence/paragraph into two. That's the standard formatting for this kind of thing.)

Comment (Riolku) [reply testsuite/rust-bridged/README.md]: Done.

**Review by Xyene — APPROVED** [review]
LGTM, thanks!

**Pull request merged**